web_client/stylesheets/dialogs/editStyleGroups.styl

```diff
@@ -0,0 +1,4 @@
+@import "~nib/index.styl"
```

Comment:
I thought we changed Girder so that the webpack builder always adds ~nib/index.styl to our `styl` files.